app/src/main/java/se/kth/sda/othello/imp/BoardImp.java

```diff
@@ -12,13 +13,21 @@
 public class BoardImp implements Board {
     Node nodes[][] = new Node[8][8];
 
+    ArrayList<Node> initialNodes = new ArrayList<>();
+
     // TODO: Correctly initialize the four inital nodes
     public BoardImp() {
         for (int i=0; i<8; i++) {
             for (int j=0; j<8; j++) {
                 nodes[i][j] = new NodeImp(i,j);
             }
         }
+        initialNodes.add(new NodeImp(3,3));
```

Comment:
There are some wrongs about the initial 4 central coins. 
Because you add the implement coding in BoardImp.
BoardImp  just does something logic about board, not for nodes.
Please add the implementation in OthelloImp. You can call the method setNode in the BoardImp.
If you want to see how to use setNode method , see public List<Node> move(String playerId, String nodeId) throws IllegalStateException

Reply:
Can't agree.
Well, first of all, Roberto specified the initialization of the central nodes by a TODO comment, which is still in the code.
And I also think that the nodes should initialized in the board, not in the game. A node belongs to a board, not to a game.

Reply:
In my opinion， it should add in the method start() in OthelloImp,
 @Override
    public void start() {
        // TODO: choose a random player
        currentPlayer = playerOne;
    }

    @Override
    public void start(String playerId) {
        // TODO: choose the player with the right ID
        currentPlayer = playerOne;
    }
In the start() method, set the current player, and all the initial about the game.

But if you want to keep it in the BoardImp, please add playerId for them.